### PR TITLE
Add image overlay

### PR DESCRIPTION
**Pull request by seekuehe**

### CODE REVIEW

File: src/gen-xml.ts [hunk 1]
```diff
@@ -565,7 +565,16 @@ function slideObjectToXml(slide: PresSlide | SlideLayout): string {
 				let sizing = slideItemObj.options.sizing,
 					rounding = slideItemObj.options.rounding,
 					width = cx,
-					height = cy
+					height = cy,
+					overlayXmlString = slideItemObj.options.overlay
```
Comment (tjinauyeung) [inline src/gen-xml.ts]: How about extracting this to a function e.g. `genXmlOverlay()`

Comment (tjinauyeung) [reply src/gen-xml.ts]: We should validate the `overlay` options to prevent ending up with a corrupted pptx

Comment (tjinauyeung) [reply src/gen-xml.ts]: I suggest having something similar to `correctShadowOptions` for overlay options, which checks if the passed overlay options are valid and sets defaults + console warning if they're not.

**Comment by pastafari** [comment]
Summarizing what we discussed:
- lets call the thing what its called in OOXML, so `solidFill` rather than overlay.

**Review by tjinauyeung — APPROVED** [review]
💯

**Pull request merged**